src/app/ComputerPlayer.js

```diff
@@ -0,0 +1,12 @@
+export class ComputerPlayer {
+    constructor(answerSelectedCallback) {
+        this.answerSelectedCallback = answerSelectedCallback;
+    }
+    askQuestion(answersList) {
+        const selectedAnswerIndex = Math.floor(Math.random() * 4);
```

Comment:
Pod tą czwórkę powinniśmy mieć zmienną w constants, żeby gdyby w przyszłości okazało się że jest to inna liczba odpowiedzi to nie szukać i poprawiac po czałym kodzie

Reply:
a może zamiast czwórki - answersList.length? bo chyba zawsze będziemy losować z listy dostępnych odpowiedzi